docs/scripting/resources/macros.md

```diff
@@ -0,0 +1,383 @@
+---
+title: Limits
```

Comment:
Does not match the file name.

However, I propose changing this to `constants` rather than `macros`.

Reply:
On it